xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/extension/extension.css

```diff
@@ -475,3 +475,83 @@ ul.collapsible.document-tree {
   font-weight: normal;
   padding-right: .3em;
 }
+
+@media screen and (min-width: 675px) {
+    .extension-header .extension-status {
+        margin: 0 1em;
+    }
+  .dependency-item{
+       height: 7.407vw;
+  }
+  .dependency-item .extension-status {
+       padding-top: 2.962vw;
+  }
+}
+
+@media screen and (min-width: 620px) {
+    .extension-header .extension-status {
+        margin: 0 1em;
```

Comment:
A lot of duplication of the selectors and properties. The properties can be inherited from smaller resolutions ot bigger ones.